### ISSUE TITLE
Does not compile, as is

### ISSUE THREAD

**Issue by Peter0x44** [raysan5/rguistyler#5]
I tried building rguistyler just now, and got the following errors:

```c
rguistyler.c:741:26: error: ‘showLoadFileDialog’ undeclared (first use in this function); did you mean ‘showLoadFontDialog’?
  741 |                 else if (showLoadFileDialog) showLoadFileDialog = false;
      |                          ^~~~~~~~~~~~~~~~~~
      |                          showLoadFontDialog
rguistyler.c:741:26: note: each undeclared identifier is reported only once for each function it appears in
rguistyler.c:742:26: error: ‘showSaveFileDialog’ undeclared (first use in this function); did you mean ‘showSaveStyleDialog’?
  742 |                 else if (showSaveFileDialog) showSaveFileDialog = false;
      |                          ^~~~~~~~~~~~~~~~~~
      |                          showSaveStyleDialog
rguistyler.c:743:26: error: ‘showExportFileDialog’ undeclared (first use in this function); did you mean ‘showExportStyleDialog’?
  743 |                 else if (showExportFileDialog) showExportFileDialog = false;
      |                          ^~~~~~~~~~~~~~~~~~~~
      |                          showExportStyleDialog
```

Adding definitions for these global variables likeso did work:

```patch
 #include "raylib.h"
 
+bool showLoadFileDialog;
+bool showSaveFileDialog;
+bool showExportFileDialog;
```

Also, the compilation commands in the comments of rguistyler.c don't work, even with these definitions
```
       gcc -o rguistyler rguistyler.c external/tinyfiledialogs.c -s -no-pie -D_DEFAULT_SOURCE \
           -lraylib -lGL -lm -lpthread -ldl -lrt -lX11
```
They are missing `-I external`, otherwise the include of raygui.h will fail

I also think the commands should have -D PLATFORM_DESKTOP, I can't see any define of that in rguistyler.c itself
It causes a few extra warnings and also the command line support is behind this #ifdef

Also: is there a good reason to add `-fno-pie` to the command? I really don't think it should be necessary...
I would suggest adding `-Os` to the commands, if minimum executable size is desired here

**Comment by raysan5** [comment]
@Peter0x44 I've been working `rGuiStyler` lately with several redesigns, please, note this version is a WIP at the moment...
